packages/server/package.json

```diff
@@ -132,16 +122,17 @@
     "package.json",
     "adapters",
     "http",
-    "rpc",
     "observable",
-    "subscription",
+    "rpc",
     "shared",
-    "unstableInternalsExport",
     "!**/*.test.*"
   ],
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "@trpc/core": "10.45.0"
```

Comment:
idk what happens if multiple package bundles the same dep, does the code get like duplicated? is that why you only have it bundled in server package and the rest are just peers?

i almost wish we could remove the peerDep of server in the rest of the packages and only depend on core?